src/ride/coaster/virginia_reel.c

```diff
@@ -56,14 +70,9 @@ void vehicle_visual_virginia_reel(int x, int imageDirection, int y, int z, rct_v
 	}
 	baseImage_id += vehicleEntry->base_image_id;
 
-	sint16 bbo_x = RCT2_ADDRESS(0x009927E6, sint8)[j * 8];
-	sint16 bbo_y = RCT2_ADDRESS(0x009927E7, sint8)[j * 8];
-	sint16 bbo_z = RCT2_ADDRESS(0x009927E8, sint8)[j * 8] + z;
-	uint16 bbl_x = RCT2_ADDRESS(0x009927E9, uint8)[j * 8];
-	uint16 bbl_y = RCT2_ADDRESS(0x009927EA, uint8)[j * 8];
-	uint8 bbl_z = RCT2_ADDRESS(0x009927EB, uint8)[j * 8];
+	vehicle_boundbox bb = _virginiaReelBoundbox[j];
```

Comment:
Ideally we should be using a pointer to the bound box structure to save copying this data about.